eclipse-scout-core/src/tabbox/SimpleTab.ts

```diff
@@ -85,6 +85,9 @@ export class SimpleTab<TView extends SimpleTabView = SimpleTabView> extends Widg
     this.saveNeeded = (this.view ? this.view.saveNeeded : model.saveNeeded);
     this.saveNeededVisible = (this.view ? this.view.saveNeededVisible : model.saveNeededVisible);
     this.status = (this.view ? this.view.status : model.status);
+    this.uuid = model.uuid;
+    this.classId = model.classId;
```

Comment:
Why is this the only widget that does this manually? Isn't this already done by the super call?